### DISCUSSION TITLE
Wallet transactions and balance are not displayed right in wallet

### DISCUSSION THREAD

**Discussion by 17und4** [Chia-Network/chia-blockchain#6524] [Support]
Hi,

I have a serious problem with my wallet, it is not correctly displaying the transactions, incoming vs outgoing in addition the coin count in my wallet is also not correct. How can I fix this?

Thanks

Manfred

**Comment by loppefaaret** [comment]
You could nuke the wallet database, and resync ot from scratch:
Close down Chia app, and open up a PowerShell window, and execute this command:
`del ~\.chia\mainnet\wallet\db\block*`
This removes any wallet database completely, you can then open up Chia app again, and wallet should start syncing from 0

**Comment by BrandtH22** [comment]
@17und4 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
